stable/prometheus-redis-exporter/values.yaml

```diff
@@ -9,3 +9,7 @@ service:
   annotations: {}
 resources: {}
 redisAddress: redis://myredis:6379
+annotations: {}
```

Comment:
Hi @Juanchimienti 

Should this replace the service annotations?

Reply:
not really, prometheus scrapes the pods directly

Reply:
Got it, looks good